src/ast/ddl.rs

```diff
@@ -428,6 +474,19 @@ fn display_constraint_name(name: &'_ Option<Ident>) -> impl fmt::Display + '_ {
     ConstraintName(name)
 }
 
+fn display_key_name(name: &'_ Option<Ident>) -> impl fmt::Display + '_ {
```

Comment:
Too much complex, this isn't necessary at all...

Please can you try to make it simple? LMK if you need any help

Reply:
It's similar to display_constraint_name, the name dose not implement display method, so i think it's necessary to do this.